Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,33 @@
-function setAlarm() {}
+function setAlarm() {
+  const alarmSet = document.getElementById("alarmSet").value;
+  const timeRemaining = document.getElementById("timeRemaining");
+
+  let totalSeconds = parseInt(alarmSet);
+
+  let countdown = setInterval(() => {
+    let minutes = Math.floor(totalSeconds / 60);
+    let seconds = totalSeconds % 60;
+
+    timeRemaining.textContent = `Time Remaining: ${String(
+      minutes
+    ).padStart(2, "0")}:${String(seconds).padStart(2, "0")}`;
+
+    if (totalSeconds <= 0) {
+      clearInterval(countdown);
+      playAlarm();
+      let flash = setInterval(() => {
+        document.body.style.backgroundColor =
+          document.body.style.backgroundColor === "red" ? "white" : "red";
+      }, 500);
+      setTimeout(() => {
+        clearInterval(flash);
```

Comment:
Nice job stopping the flashing after a while - this wasn't a requirement of the task, but is a nice (and calming) effect.